src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -91,46 +192,90 @@ export default function Inbox () {
   </PrimaryLayout>
 }
 
-export function TransactionRow ({ transaction, showConfirmationModal }) {
-  const { counterparty, amount, type, timestamp, notes } = transaction
+export function TransactionRow ({ transaction, actionsClickWithTx, actionsVisible, showConfirmationModal, inboxView, whoami }) {
+  const { counterparty, presentBalance, amount, type, notes } = transaction
+
+  const actionsClick = () => actionsClickWithTx(transaction)
+  const handleCloseReveal = () => {
+    if (!isEmpty(actionsVisible) && actionsVisible === transaction) return actionsClickWithTx(null)
+    else if (!isEmpty(actionsVisible) && actionsVisible !== transaction) return actionsClickWithTx(transaction)
+    else return actionsClickWithTx(null)
+  }
+
+  let agent
+  if (counterparty.id === whoami.id) agent = whoami
+  else agent = counterparty
 
   const isOffer = type === TYPE.offer
   const isRequest = !isOffer
 
-  const { date, time } = presentDateAndTime(timestamp)
-
-  const story = isOffer ? ' is offering' : ' is requesting'
+  let story
+  if (inboxView === VIEW.pending) story = isOffer ? ' is offering' : ' is requesting'
```

Comment:
rather than having the TransactionRow care about the state of the inbox, it makes more sense to me that the TransactionRow just knows whether this is a pending transaction or not. I think we can even get that from the transaction object, so no need to pass in inboxView.

I can also see an argument for wanting the parent to be in control of whether this displays as pending, rather than the TransactionRow, in which case you can just pass in

`isPending={inboxView === View.pending}`
or
`isPending={isPending(transaction)}`      // this test could also be done in TransactionRow, I feel mixed about that.
or whatever other condition, as a prop to `TransactionRow`

This is the same principle that motivated my request around actionsVisible: right now `TransactionRow` knows more than it needs to about the inner workings of the Inbox component, which makes it harder to refactor/maintain/change.